packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -413,6 +418,7 @@ const SelectInput = (props: TSelectInputProps) => {
           maxMenuHeight={props.maxMenuHeight}
           menuPortalTarget={props.menuPortalTarget}
           menuShouldBlockScroll={props.menuShouldBlockScroll}
+          closeMenuOnSelect={props.closeMenuOnSelect}
```

Comment:
I think it would be great to also add the default value to the defaultProps attribute of the component

Reply:
Any particular reason for having a default value? Since we forward this prop to react-select, I would just let react-select figure out the default value, as we don't want to change that behavior.

Reply:
My reasoning was based on that, up until now, the uses cases where we use these components were relying on the menu actually closing upon selection and, given that now we can select that behaviour, we would expect it to keep it that way by default so we don't break anything. I mean, it seems to me that our default is closing the menu upon selection.
If we don't use our own default and rely on `react-select` and the library's author decides to change its default behaviour, our own use cases would change as well but not because it was our intention but the library's author.

Anyway, that was the way I thought about this but it's ok from my side if we prefer not to use this default.

Reply:
It makes sense from @CarlosCortizasCT point of view to have a default value, however, should I keep or remove the default value? @emmenko

Reply:
I understand but we would need to potentially do this for every prop that we forward, which becomes a bit unmaintainable.

If react-select changes the default behavior, it would be a breaking change so we can then adjust things accordingly from our side.

So I would prefer to not have default values from our side for now.